### DISCUSSION TITLE
What are the best C++ libraries for parsing for people with an intelligence delay.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#120288] [Programming Help]
### Body

What are the best C++ libraries for people with an intelligence delay.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by B3TA-BLOCKER** [comment]
SFML
```cpp
    #include <SFML/Graphics.hpp>
```

OpenCV (for displaying and loading images)
```cpp
   #include <opencv2/opencv.hpp>
```

Qt (fro creating simple window)
```cpp
   #include <QtWidgets/QApplication>
   #include <QtWidgets/QMainWindow>
```

**Comment by tstran155** [comment]
Hi  PyDOS8,

Some general suggestions for C++ libraries and resources that might be more accessible:

- Poco (for building internet and network applications)
```
#include <Poco/Net/HTTPClientSession.h>
#include <Poco/Net/HTTPRequest.h>
#include <Poco/Net/HTTPResponse.h>
#include <Poco/StreamCopier.h>
```
- Boost (a collection of portable and well-tested libraries and components that only require a C++ Standard-compliant compiler)
```
#include <boost/filesystem.hpp>
```
- Qt (for developing graphical user interfaces (GUIs))
```
#include <QApplication>
#include <QWidget>
```
- SFML (for simple interface to various components of multimedia programming for 2D game development or applications)
```
#include <SFML/Graphics.hpp>
```

**Reply by ghost** [reply]
Hey [tstran155](https://github.com/tstran155),

Thanks for the libraries I'll check them out later.
